src/js/components/AccordionPanel.js

```diff
@@ -0,0 +1,74 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Box from './Box';
+import Heading from './Heading';
+import ListItem from './ListItem';
+import OpenIcon from './icons/base/Add';
+import CloseIcon from './icons/base/Subtract';
+import Collapsible from './Collapsible';
+
+import CSSClassnames from '../utils/CSSClassnames';
+
+const CLASS_ROOT = CSSClassnames.ACCORDION_PANEL;
+
+export default class AccordionPanel extends Component {
+  constructor (props) {
+    super(props);
+    this._onClickPanel = this._onClickPanel.bind(this);
+    this.state = {
+      isOpen: false
+    };
+  }
+
+  _onClickPanel () {
+    this.setState({ isOpen : !this.state.isOpen });
+  }
+
+  render () {
+    const { animate, title, children } = this.props;
+
+    const classes = classnames(
+      CLASS_ROOT,
+      this.props.className,
+      {
+        [`${CLASS_ROOT}--active`]: this.state.isOpen
+      }
+    );
+
+    let panelControlIcon = <OpenIcon colorIndex="brand" />;
```

Comment:
Most of our other controls are not brand colored. I think we should review whether these are as well with Chris.